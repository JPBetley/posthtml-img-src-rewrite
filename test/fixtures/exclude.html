<!doctype html>
<img src="images/test.png">
<img src="images/exclude.png" class="exclude">
