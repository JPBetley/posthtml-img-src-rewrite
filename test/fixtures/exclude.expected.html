<!doctype html>
<img src="images/prefix_test_suffix.png">
<img src="images/exclude.png" class="exclude">
